src/include/verbose.h

```diff
@@ -134,6 +134,27 @@ struct __wt_verbose_multi_category {
 #define WT_VERBOSE_ISSET(session, category) \
     WT_VERBOSE_LEVEL_ISSET(session, category, WT_VERBOSE_LEVEL_DEFAULT)
 
+/* get the verbosity level for a given category. */
+#define WT_GET_VERBOSE_LEVEL(session, category, ret)                         \
+    do {                                                                     \
+        ret = S2C(session)->verbose[category];                               \
```

Comment:
WiredTiger only uses `ret` for tracking return codes. Could you change this to a different name please?